pkg/synthetictests/kubelet.go

```diff
@@ -16,6 +16,11 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+const (
+	probeTimeoutEventThreshold                 = 5
+	probeTimeoutConfigOperatorMessageRegExpStr = "reason/ReadinessFailed Get.*healthz.*net/http.*request canceled while waiting for connection.*Client.Timeout exceeded"
```

Comment:
The variable name says config operator but this looks like it'd match anything -- shouldn't it include ns/openshift-config-operator in the regex?

Reply:
(I realize it's checked below, but duplicated_events.go uses this to filter out events, so it will filter out all ReadinessFailed events from everything.)

Reply:
Looks like regex is still the same

Reply:
in the test, I look for the openshift-config-operator pod in the openshift-config-operator namespace in particular.  If we wanted to make another test for another operator, we can do something similar and keep the same regex.

Reply:
You reference this regexp here:

https://github.com/openshift/origin/pull/27436/files#diff-d0d69df50187ca4d15b52658177839bafded2d8437096c4161f148b1f5f178caR126-R127

You will filter ALL matching records, thus effectively turning off reporting for any of them except the openshift-config-operator

Reply:
ah, thanks for pointing that out, again :-). I just noticed you mentioned it earlier and I didn't even notice -- my oversight.  I added the pod/container into the generated message we put into the event so that it can be more specific to the pod/container we care about in tests.  Then I can make the filter more specific.